tests/UnitTestCase.php

```diff
@@ -0,0 +1,159 @@
+<?php
+declare(strict_types=1);
+
+namespace Yiisoft\Queue\Redis\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Psr\Log\NullLogger;
+use Yiisoft\Injector\Injector;
+use Yiisoft\Queue\Adapter\AdapterInterface;
+use Yiisoft\Queue\Cli\LoopInterface;
+use Yiisoft\Queue\Cli\SignalLoop;
+use Yiisoft\Queue\Message\JsonMessageSerializer;
+use Yiisoft\Queue\Message\MessageInterface;
+use Yiisoft\Queue\Middleware\CallableFactory;
+use Yiisoft\Queue\Middleware\Consume\ConsumeMiddlewareDispatcher;
+use Yiisoft\Queue\Middleware\Consume\MiddlewareFactoryConsume;
+use Yiisoft\Queue\Middleware\FailureHandling\FailureMiddlewareDispatcher;
+use Yiisoft\Queue\Middleware\FailureHandling\MiddlewareFactoryFailure;
+use Yiisoft\Queue\Middleware\Push\MiddlewareFactoryPush;
+use Yiisoft\Queue\Middleware\Push\PushMiddlewareDispatcher;
+use Yiisoft\Queue\Queue;
+use Yiisoft\Queue\Redis\Adapter;
+use Yiisoft\Queue\Redis\QueueProvider;
+use Yiisoft\Queue\Redis\Tests\Support\ExtendedSimpleMessageHandler;
+use Yiisoft\Queue\Redis\Tests\Support\FileHelper;
+use Yiisoft\Queue\Worker\Worker;
+use Yiisoft\Queue\Worker\WorkerInterface;
+use Yiisoft\Test\Support\Container\SimpleContainer;
+
+/**
+ * Test case for unit tests
+ */
+abstract class UnitTestCase extends TestCase
```

Comment:
Doesn't look like a unit test while redis client is used

Reply:
Yes, you are right, i was just lazy and implemented everything in unit tests) I will transfer the work with redis client to `Integration` directory.

Reply:
Done. All tests that work directly with the client redis have been moved to integration tests.